modules/UI/UI.js

```diff
@@ -712,6 +712,14 @@ UI.setVideoMuted = function (id, muted) {
 };
 
 /**
+ * Triggers an update of remote video and large video displays so they may pick
+ * up any state changes that have occurred elsewhere.
+ *
+ * @returns {void}
+ */
+UI.updateAllVideos = () => VideoLayout.updateAllVideos();
```

Comment:
That decision was likely made because the rest of the file follows that pattern. But also updateAllVideos currently has a reference to "this" so it'll need to be called somehow with the VideoLayout object, like with a VideoLayout.updateAllVideos.bind(VideoLayout).